scannermsbuild.properties

```diff
@@ -16,6 +16,15 @@ flavors.core20.label=.NET Core 2.1
 flavors.core31.label=.NET Core 3.1
 flavors.framework46.label=.NET Framework 4.6+
 
+5.10.0.description=FIPS Compliance
```

Comment:
Do we really claim FIPS compliance. Maybe we should have a  more nuanced position: Better FIPS compatibility / compliance?

Reply:
Updated, thanks